tasks/libs/common/git.py

```diff
@@ -45,6 +45,40 @@ def get_staged_files(ctx, commit="HEAD", include_deleted_files=False) -> Iterabl
                 yield file
 
 
+def get_file_modifications(
+    ctx, base_branch=DEFAULT_BRANCH, added=False, modified=False, removed=False, only_names=False
```

Comment:
That will probably not work as we expect it on release branches. Maybe you could add something like that to make sure we have the correct branch on release branches? https://github.com/DataDog/datadog-agent/blob/b7383882501637956482b60e1e112963664e74a8/tasks/gotest.py#L830